source/elements/oneDAL/source/glossary.rst

```diff
@@ -0,0 +1,277 @@
+        :term:`ordinal<Ordinal feature>` values. This is an output in
+        :term:`classification` and :term:`clustering` tasks.
+
+        **Example:** the spam-detection task has a binary label indicating whether
+        the email is spam or not
+
+    Model
+        An entiry which stores an information necessary to run :term:`inference`
+        on a new :term:`dataset`. Typically a result of :term:`training`
+        process.
+
+        **Example:** in linear regression algorithm, the model contains weight
+        values for each input feature and single bias value
+
+    Observation
+        A :term:`feature vector` and zero or more :term:`responses<Response>`.
+        Each of them can encode different objects or events.
+
+        **Synonyms:** instance, sample
+
+    Ordinal feature
+        A :term:`categorical feature` with defined operations of equality an
+        ordering between values.
+
+        **Example:** student's grade
+
+    Ratio feature
+        A :term:`continuous feature` with defined operations of equality,
+        comparison, addition, substraction, multiplication and division.
+        Has a zero value element which means the absence of value.
+
+        **Example:** the height of a tower
+
+    Regression
+        A :term:`supervised machine-learning problem<Supervised learning>` of
+        assigning :term:`continuous<Continuous feature>`
+        :term:`responses<Response>` to :term:`feature vectors<Feature vector>`.
+
+        **Example:** prediction of the temperature based on weather conditions
+
+    Response
+        A property of some real object or event which value is target to be
+        predicted in the :term:`supervised learning` task. While the
+        :term:`feature` is an input in the machine learning task, the response
+        is one of the outputs can be made by the :term:`model` on the
+        :term:`inference` stage.
+
+        **Synonym:** dependent variable
+
+    Supervised learning
+        :term:`Training` process with a :term:`dataset` containing information
+        on dependencies between :term:`features<Feature>` and
+        :term:`responses<Response>`. The goal is to get a :term:`model` of
+        dependencies between input :term:`feature vector` and
+        :term:`responses<Response>`.
+
+    Training
+        A process of creating a :term:`model` based on information extracted
+        from a :term:`training set`. Resulting :term:`model` is selected in
+        accordance with some quality criteria.
+
+    Training set
+        A :term:`dataset` used at the :term:`training` stage to create a
+        :term:`model`.
+
+    Unsupervised learning
+        :term:`Training` process where :term:`training set` does not contain
+        information on dependencies between :term:`feature vectors<Feature
+        vector>` and :term:`responses<Response>`. The goal is to get a
+        :term:`model` matching :term:`feature vectors<Feature vector>` into
+        groups.
```

Comment:
Not sure about dependencies. We don't have responses in unsupervised learning, right?
My suggestion:
```suggestion
        :term:`Training` process that uses a :term:`training set` with no
        :term:`responses <Response>`. The goal is to get a
        :term:`model` that allocates :term:`feature vectors <Feature vector>`  into
        groups.
```

Reply:
Good question about responses. At the training stage yes, we do not have them. But how to refer the outputs of the unsupervised task made at the inference stage? In my opinion, response can be used for supervised (regression+classification) tasks and unsupervised tasks. Label can be used for classification and clustering(unsupervised) tasks.
@rlnx , @andreynv , what is your opinion, can we use `response` for unsupervised tasks?

Reply:
Unsupervised learning is about the case when we have feature vectors without responses and want to find some hidden patterns, e.g. how feature vectors are 'similar'. How we represent the outcome of this learning is secondary topic, e.g., we numerate the groups of the similar feature vectors or we have a set of rules that indicate connection between individual items/features in the data (if I buy this thing, I'll highly likely buy that thing)

Reply:
Ok, thanks.